frontend/test/metabase/scenarios/question/question-management.cy.spec.js

```diff
@@ -153,6 +165,38 @@ describe("managing question from the question's details sidebar", () => {
               cy.findByText("This question has been archived");
             });
 
+            it("should show a warning when archiving a question used for dropdown widgets", () => {
```

Comment:
Would it make sense to put this test into `dashboard-filters-source`? This is essentially the same feature.